src/Routes/ImageManager/ImageTable.js

```diff
@@ -33,7 +34,14 @@ import {
 } from './constants';
 import PropTypes from 'prop-types';
 
-const ImageTable = ({ filters, pagination, openWizard, clearFilters }) => {
+const ImageTable = ({
+  filters,
+  pagination,
+  clearFilters,
+  openCreateWizard,
+  openUpdateWizard,
+  setUpdateImageID,
```

Comment:
can we create openUpdateWizard that will both open the update wizard and set the update image ID?